src/adapters/sekindoUM.js

```diff
@@ -0,0 +1,110 @@
+      //bidmanager.pbCallbackMap[callbackId] = bidReqeust;
+    }
+  }
+
+  $$PREBID_GLOBAL$$.sekindoCB = function(callbackId, response) {
+    var bidObj = getBidRequest(callbackId);
+    if (typeof (response) !== 'undefined' && typeof (response.cpm) !== 'undefined') {
+      var bid = [];
+      if (bidObj) {
+        var bidCode = bidObj.bidder;
+        var placementCode = bidObj.placementCode;
+
+        if (response.cpm !== undefined && response.cpm > 0) {
+
+          bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
+          bid.callback_uid = callbackId;
+          bid.bidderCode = bidCode;
+          bid.creative_id = response.adId;
+          bid.cpm = parseFloat(response.cpm);
+          bid.ad = response.ad;
+          bid.width = response.width;
+          bid.height = response.height;
+
+          bidmanager.addBidResponse(placementCode, bid);
+        }
+
+        else {
+          bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
+          bid.callback_uid = callbackId;
+          bid.bidderCode = bidCode;
+          bidmanager.addBidResponse(placementCode, bid);
+        }
+      }
+    }
+
+    else {
+      if (bidObj) {
+        utils.logMessage('No prebid response for placement '+bidObj.placementCode);
+      }
+
+      else {
+        utils.logMessage('sekindoUM callback general error');
+      }
+    }
+  };
+
+  function _requestBids(bid, callbackId, pubUrl) {
+    //determine tag params
+    var spaceId = utils.getBidIdParameter('spaceId', bid.params);
+    var subId = utils.getBidIdParameter('subId', bid.params);
+    var bidfloor = utils.getBidIdParameter('bidfloor', bid.params);
+    var protocol = ('https:' === document.location.protocol ? 's' : '');
+    var scriptSrc = 'http'+protocol+'://hb.sekindo.com/live/liveView.php?';
+
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 's', spaceId);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'subId', subId);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'pubUrl', pubUrl);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'hbcb', callbackId);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'hbver', '2');
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'hbobj', '$$PREBID_GLOBAL$$');
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'dcpmflr', bidfloor);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'hbto', $$PREBID_GLOBAL$$.bidderTimeout);
+    scriptSrc = utils.tryAppendQueryString(scriptSrc, 'protocol', protocol);
+
+    var html = '<scr'+'ipt type="text/javascript" src="'+scriptSrc+'"></scr'+'ipt>';
+
+    var iframe = utils.createInvisibleIframe();
+    iframe.id = 'skIfr_'+callbackId;
+
+    var elToAppend = document.getElementsByTagName('head')[0];
+    //insert the iframe into document
+    elToAppend.insertBefore(iframe, elToAppend.firstChild);
+
+    var iframeDoc = utils.getIframeDocument(iframe);
+    iframeDoc.open();
+    iframeDoc.write(html);
```

Comment:
why are you writing your script into an iframe? It doesn't sandbox it because you are doc.writing it instead of using `iframe src=`.  Might as well add it to the current window to save the overhead of creating an iframe.